### ISSUE TITLE
Optional password protection

### ISSUE THREAD

**Issue by fumbleforce** [fabienb4/meteor-jsdoc#29]
Say you want to host your documentation online, but as it is a commercial project, you only want people with access to your project to be able to access it.

It would then be nice to have a simple authentication front where you had to provide a password to see the documentation page.

This could be done with the accounts-password package, where an admin user is created automatically, and the password was gotten from the settings file.

**Comment by fabienb4** [comment]
Hey @jorgeer, sorry for the delayed answer.

This can be added on the docs side (after `meteor-jsdoc build`).
You need to setup the config properly (disable `initMeteor` only after the first build):

``` json
"initMeteor": false,
"updateMeteor": false
```

And then you can modify the meteor project (the docs' one) however you like.
Add you authentication package (`accounts-ui`), and setup the template to show the `docs` only if logged in.
Example (in `client/head.html`):

``` diff
<body>
-  {{> docs}}
+  {{#if currentUser}}
+    {{> docs}}
+  {{else}}
+    {{> loginButtons}}
+  {{/if}}
</body>
```

You can then create the users you want and disable registrations.

``` js
Accounts.validateNewUser(function() {
  return false;
});
```

Again, maybe due to my strong opinions, but I don't think it is the job of `meteor-jsdoc` to handle users, especially if you want proper security (change password and all).
I try to keep this package as simple as possible, but offer complete customization if people needs it.
You can see that the above is pretty straight forward, and simple enough to be implemented on a per project basis, instead of in `meteor-jsdoc`.
